core/include/mm/tee_mmu_types.h

```diff
@@ -29,6 +30,12 @@
 
 #define TEE_MATTR_GLOBAL		BIT(10)
 #define TEE_MATTR_SECURE		BIT(11)
+#define TEE_MATTR_DEVICE		BIT(13)
```

Comment:
Wouldn't it make more sense to group this device stuff as different kinds of cache settings?
If you don't want to call it `TEE_MATTR_CACHE*` we could rename it to `TEE_MATTR_MEM_TYPE*`

Reply:
ACK